docs/root/configuration/security/secret.rst

```diff
@@ -15,7 +15,10 @@ Upstream clusters are handled in a similar way, if a cluster client certificate
 
 If a static cluster is using SDS, and it needs to define a SDS cluster (unless Google gRPC is used which doesn't need a cluster), the SDS cluster has to be defined before the static clusters using it.
 
-The connection between Envoy proxy and SDS server has to be secure. One option is to run the SDS server on the same host and use Unix Domain Socket for the connection. Otherwise it requires mTLS between the proxy and SDS server. In this case, the client certificates for the SDS connection must be statically configured.
+The connection between Envoy proxy and SDS server has to be secure. One option is to run the SDS server on the same host and use Unix Domain Socket for the connection. Otherwise it requires valid authentication between the proxy and SDS server using gRPC channel credentials, call credentials, or both. Credential types in use today are:
```

Comment:
Are we confusing authentication and confidentiality here? There's two aspects:
1. We need to ensure that there is always server TLS configured, i.e. an upstream TLS context for Envoy gRPC/REST, and not using someting like grpc::InsecureChannelCredentials() if it's the Google gRPC client.
2. We need to authenticate. This should be over a secure channel, see (1), but this is where client certs or SigV4 come into play.
The docs read as if we can solve (2) with (1) and don't cover Envoy gRPC/REST.

Reply:
Agree with both points, and will reword. :)